### ISSUE TITLE
Settings often wiped after updates

### ISSUE THREAD

**Issue by dylanh724** [ShareX/ShareX#6498]
I reported this [last year](https://github.com/ShareX/ShareX/issues/5890), but I wanted to follow-up on this critical bug: Upon an update, the settings wipe.

Update pops up -> update -> sharex returns -> settings are gone. Repro'd on both my wife's laptop, my desktop, my laptop (x3).

* Windows 11 - but this happened in 10, too.
* ...there's not really much more to add - nothing special. All versions do this upon updating; but not every time. Seemingly inadvertent.

I tried to search the issue, but surprisingly couldn't find much. Any other info you folks need?

**Comment by dylanh724** [comment]
Fyi: Reported this [over a year ago](https://github.com/ShareX/ShareX/issues/5890).

**Comment by dylanh724** [comment]
Hey if it helps, I actually downloaded this with [Ninite](https://ninite.com/) both times - could it save settings in different places?

**Comment by wolfsprite** [comment]
I've downloaded it natively on multiple machines, and everytime I've updated, the settings get wiped, so it is not exclusive to Ninite intallation. 

This has been reported multiple times since 2016 and has sadly yet to be fixed.

As I posted earlier today after getting updated to a reply in this other related (sadly closed) issue, my solution has sadly to just never update ShareX. :/ It works as a screenshot tool, and updating causes too much headaches in terms of having to set up all preferences again.

https://github.com/ShareX/ShareX/issues/1857#issuecomment-1528011006

Really hope someone actually fixes this someday, because it's quite a blemish on an otherwise excellent software.
